Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.cpp

```diff
@@ -105,22 +105,22 @@ namespace AzPhysics
         }
     }
 
-    constexpr AZStd::string_view KinematicDescription =
-        "When active, the rigid body is not affected by gravity or other forces and is moved by script. ";
+    static constexpr AZStd::string_view KinematicDescription =
+        "Determines how the movement / position of the rigid body is controlled.";
```

Comment:
uber nit: I'd omit the space around the `/` character (so just `movement/position`) - repeated below